SECURITY.md

```diff
@@ -19,6 +19,7 @@ LoopBack implements the
 **Database connectors**
 
 - [Cassandra connector](https://loopback.io/doc/en/lb4/Cassandra-connector.html)
+- [CouchDB 2.x connector](https://loopback.io/doc/en/lb4/Couchdb2-connector.html)
```

Comment:
on a second thought, i'm thinking whether it's better to just have `CouchDB connector`, so that it's consistent with the other connector names.